mote/templates/mote/home.html

```diff
@@ -1,17 +1,69 @@
 {% extends "mote/base.html" %}
 
 {% block hero %}
-    <div class="background"></div>
-    <div class="foreground">
-        <h1>Pattern Library</h1>
+    <div class="Hero-background">
+        <svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
It's the atom icon that sits behind the text. Didn't have time to make it DRY - that'll come later.